srai/joiners/intersection_joiner.py

```diff
@@ -98,7 +98,7 @@ def _join_without_geom(
         joint = (
             gpd.sjoin(
                 regions.reset_index(names=REGIONS_INDEX),
-                features.reset_index(names=FEATURES_INDEX),
+                features[["geometry"]].reset_index(names=FEATURES_INDEX),
```

Comment:
Maybe use `GEOMETRY_COLUMN` from `srai.constants` instead of string "geometry".

Reply:
It doesn't exist yet (the change is on different PR), but I will add it here

Reply:
It's in my PR, my bad 😄